add printer ThickInk mark 1

guichiang opened the pull request:
ThickInk is a 3d printer with 4 extruders

fieldOfView commented:
Your files are fairly inconsistent in their use of tabs vs spaces. Could you make them all use 4 space instead of tabs?

nallath commented:
It also has quite a few mistakes with it. It's referring to non-existing platform mesh, there are a number of default values that don't need to be there and the machine_ids inside the extruder don't match.

guichiang commented:
ook, I corrected using  4 space. I used other machine definitions to
understand how to write the code.

There are same errors that I didn't understand what is and how to solve it.

[image: image.png]
1. This first I don't know what I need to do.

[image: image.png]
2. These I forgot to upload the platform STL, I suppose this is the error.


[image: image.png]
3. I tried to understand what is wrong but I don't get it. It says about
definitions containers "default_value" that I used and the "fdmprinter" (I
suppose) defines a "value" already. My definition isn't do diferent
compared to other machines definitions. I search for a correct form to
write the definition and I found these 2 links and compare to my definition
I don't see what is wrong.
https://github.com/Ultimaker/Cura/blob/master/resources/definitions/fdmextruder.def.json

https://github.com/Ultimaker/Cura/blob/master/resources/definitions/fdmprinter.def.json


[image: image.png]
4. These repeated for all extruder files. The machine name needs to be the
machine file name? I think it's the problem.

When I correct the files can I send to you to verify if its is correct or I
upload again to the github?

Thanks for your attention.
Guilherme Chiang

Em sex., 5 de jun. de 2020 às 09:32, Aldo Hoeben <notifications@github.com>
escreveu:

> Your files are fairly inconsistent in their use of tabs vs spaces. Could
> you make them all use 4 space instead of tabs?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Ultimaker/Cura/pull/7881#issuecomment-639453561>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKLOZKR4ULR57JVGLTODKELRVDQXLANCNFSM4NTQLNGA>
> .
>


-- 
[image: photo]
*Guilherme Haw Wei Chiang*
Mechatronics Engineer, Unicamp, Brazil

55 11 95409-6055 | g217209@dac.unicamp.br

https://www.urubusaero.com/
<http://www.youtube.com/user/urubusaero/>
<http://www.facebook.com/UrubusAeroDesign/>
<http://www.instagram.com/urubus_aerodesign/?hl=pt-br>

guichiang commented:
I forgot to send the files that I corrected.

Em sex., 5 de jun. de 2020 às 11:58, gui chiang <guilhermechiang@gmail.com>
escreveu:

> ook, I corrected using  4 space. I used other machine definitions to
> understand how to write the code.
>
> There are same errors that I didn't understand what is and how to solve it.
>
> [image: image.png]
> 1. This first I don't know what I need to do.
>
> [image: image.png]
> 2. These I forgot to upload the platform STL, I suppose this is the error.
>
>
> [image: image.png]
> 3. I tried to understand what is wrong but I don't get it. It says about
> definitions containers "default_value" that I used and the "fdmprinter" (I
> suppose) defines a "value" already. My definition isn't do diferent
> compared to other machines definitions. I search for a correct form to
> write the definition and I found these 2 links and compare to my definition
> I don't see what is wrong.
>
> https://github.com/Ultimaker/Cura/blob/master/resources/definitions/fdmextruder.def.json
>
>
> https://github.com/Ultimaker/Cura/blob/master/resources/definitions/fdmprinter.def.json
>
>
> [image: image.png]
> 4. These repeated for all extruder files. The machine name needs to be the
> machine file name? I think it's the problem.
>
> When I correct the files can I send to you to verify if its is correct or
> I upload again to the github?
>
> Thanks for your attention.
> Guilherme Chiang
>
> Em sex., 5 de jun. de 2020 às 09:32, Aldo Hoeben <notifications@github.com>
> escreveu:
>
>> Your files are fairly inconsistent in their use of tabs vs spaces. Could
>> you make them all use 4 space instead of tabs?
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Ultimaker/Cura/pull/7881#issuecomment-639453561>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AKLOZKR4ULR57JVGLTODKELRVDQXLANCNFSM4NTQLNGA>
>> .
>>
>
>
> --
> [image: photo]
> *Guilherme Haw Wei Chiang*
> Mechatronics Engineer, Unicamp, Brazil
>
> 55 11 95409-6055 | g217209@dac.unicamp.br
>
> https://www.urubusaero.com/
> <http://www.youtube.com/user/urubusaero/>
> <http://www.facebook.com/UrubusAeroDesign/>
> <http://www.instagram.com/urubus_aerodesign/?hl=pt-br>
>


-- 
[image: photo]
*Guilherme Haw Wei Chiang*
Mechatronics Engineer, Unicamp, Brazil

55 11 95409-6055 | g217209@dac.unicamp.br

https://www.urubusaero.com/
<http://www.youtube.com/user/urubusaero/>
<http://www.facebook.com/UrubusAeroDesign/>
<http://www.instagram.com/urubus_aerodesign/?hl=pt-br>

fieldOfView commented:
Attachments in emails don't work here. You have to commit your changes and push them to your github repo.

guichiang commented:
Hey, can you help me please?

[image: image.png]

What does this error mean? Is something that I write wrong?

Em sex., 5 de jun. de 2020 às 13:17, Aldo Hoeben <notifications@github.com>
escreveu:

> Attachments in emails don't work here. You have to commit your changes and
> push them to your github repo.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Ultimaker/Cura/pull/7881#issuecomment-639605272>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKLOZKRDVK4CEM3L3Q73BXDRVELCBANCNFSM4NTQLNGA>
> .
>


-- 
[image: photo]
*Guilherme Haw Wei Chiang*
Mechatronics Engineer, Unicamp, Brazil

55 11 95409-6055 | g217209@dac.unicamp.br

https://www.urubusaero.com/
<http://www.youtube.com/user/urubusaero/>
<http://www.facebook.com/UrubusAeroDesign/>
<http://www.instagram.com/urubus_aerodesign/?hl=pt-br>

guichiang commented:
![Untitled](https://user-images.githubusercontent.com/43445418/83930155-8f421280-a76c-11ea-92c9-58640517d3ee.png)

What does this error mean? Is something that I write wrong?

fieldOfView commented:
The file you included is named `ThickInk_MK_1.STL` and the file you referenced is named `ThickInk_MK_1.stl`. The case does not match. The convention is to have lowercase extensions. Speaking of lowercase, the convention is to have lowercase filenames for your .def.json files too.

guichiang commented:
I'm getting a issue with the filename. I have change all to lowercase, but when I upload to github it's treated like a file code upload and it don't upload the file name. What can I do? remove the files and upload again?

Ghostkeeper commented:
Normally you'd use `git mv old_filename new_filename` to rename a file. But re-uploading works just as well.

konskarm commented:
I cannot find any references of this printer. Would you mind sharing a link to this printer?

konskarm commented:
Devs see CURA-7513

ninovanhooff commented:
Seems to me like this is a personal / home-built printer. It doesn't make sense to include it if other people won't be able to find, buy or build it. Closing for now, but feel free  to provide additional clarification.
